packages/odyssey-lint-staged/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "@okta/odyssey-lint-staged",
+  "version": "0.0.0",
+  "main": "src/index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "prettier": "2.4.0"
```

I think we can specify the other package you setup in the parent commit here instead of using prettier directly at 2.4.0:

```suggestion
"@okta/odyssey-prettier": "0.0.0"
```